_docs/data-model/v1/upload.md

```diff
@@ -55,6 +58,56 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+For all of the fields under device information, we are trying to provide a basic taxonomy and standards so that it will be easy to query for specific device types or classes.
+
+Thus, if you are storing information in the Tidepool platform and are unsure of what information should go here, please contact us so we can help you and maintain a high standard of data quality.
+
+### deviceTags
+This is a list of broad attributes that can be applied to the device. It is an array of individual string values. Please use exactly the attributes below if they apply, in addition to any others that are appropriate:
+
+* `"insulin-pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
+* `"fgm"` (for 'flash' glucose meters that are not continuous meters but able to read on demand)
+* `"pen"` (for insulin injection devices that record the doses)
+* `"manual"` (for manually entered data)
+
+Integrated devices or multifunction devices should use multiple tags as appropriate. For example, the Animas Vibe combo insulin delivery and CGM system would have tags `insulin-pump` and `cgm`. For devices that do not fit in these categories, please contact Tidepool so we can standardize on a good tag.
+
+### deviceManufacturer
+
+This is the name of the device manufacturer. We are trying to standardize this field, so please use exactly the following names:
```

Comment:
This *can* be an array though. `Animas` and `Dexcom` for the Animas Vibe, for ex. Want to call that out?